pycbc/events/stat.py

```diff
@@ -1779,8 +1793,16 @@ def single(self, trigs):
             The array of single detector values
         """
         from pycbc.conversions import mchirp_from_mass1_mass2
-        self.curr_mchirp = mchirp_from_mass1_mass2(trigs.param['mass1'],
-                                                   trigs.param['mass2'])
+        try:
+            self.curr_mchirp = mchirp_from_mass1_mass2(trigs.param['mass1'],
+                                                       trigs.param['mass2'])
+        except AttributeError:
+            try:
+                self.curr_mchirp = mchirp_from_mass1_mass2(trigs['mass1'],
+                                                           trigs['mass2'])
+            except KeyError:
+                logging.info("Triggers not of the correct format.")
```

Comment:
Should this be a warning / error?

Reply:
Whoops yes it should be, will fix it in the next commit